Apps/W1/Shopify/app/src/Order handling/Codeunits/ShpfyImportOrder.Codeunit.al

```diff
@@ -299,6 +299,25 @@ codeunit 30161 "Shpfy Import Order"
         end;
     end;
 
+    [NonDebuggable]
+    local procedure ImportCustomAttributtes(ShopifyOrderId: BigInteger; OrderLineId: Guid; JCustomAttributtes: JsonArray)
+    var
+        OrderLineAttribute: Record "Shpfy Order Line Attribute";
+        JToken: JsonToken;
+    begin
+        OrderLineAttribute.SetRange("Order Id", ShopifyOrderId);
+        if not OrderLineAttribute.IsEmpty then
```

Comment:
IsEmpty is procedure and should have paranetheses.

Reply:
done